README.md

```diff
@@ -184,58 +174,57 @@ When updating this, also update:
 
- * 1.30 to now - Rust 1.63
- * 1.27 to 1.29 - Rust 1.56
- * 1.17 to 1.26 - Rust 1.49
- * 1.15 to 1.16 - Rust 1.46
- * 1.0 to 1.14 - Rust 1.45
+- 1.30 to now - Rust 1.63
+- 1.27 to 1.29 - Rust 1.56
+- 1.17 to 1.26 - Rust 1.49
+- 1.15 to 1.16 - Rust 1.46
+- 1.0 to 1.14 - Rust 1.45
 
-Note that although we try to avoid the situation where a dependency transitively
-increases the MSRV of Tokio, we do not guarantee that this does not happen.
-However, every minor release will have some set of versions of dependencies that
-works with the MSRV of that minor release.
+Note that although we try to avoid the situation where a dependency transitively increases the MSRV
+of Tokio, we do not guarantee that this does not happen. However, every minor release will have some
+set of versions of dependencies that works with the MSRV of that minor release.
 
 ## Release schedule
 
-Tokio doesn't follow a fixed release schedule, but we typically make one to two
-new minor releases each month. We make patch releases for bugfixes as necessary.
+Tokio doesn't follow a fixed release schedule, but we typically make one to two new minor releases
+each month. We make patch releases for bugfixes as necessary.
 
 ## Bug patching policy
 
-For the purposes of making patch releases with bugfixes, we have designated
-certain minor releases as LTS (long term support) releases. Whenever a bug
-warrants a patch release with a fix for the bug, it will be backported and
-released as a new patch release for each LTS minor version. Our current LTS
+For the purposes of making patch releases with bugfixes, we have designated certain minor releases
+as LTS (long term support) releases. Whenever a bug warrants a patch release with a fix for the bug,
+it will be backported and released as a new patch release for each LTS minor version. Our current
+LTS
 releases are:
 
- * `1.32.x` - LTS release until September 2024. (MSRV 1.63)
- * `1.36.x` - LTS release until March 2025. (MSRV 1.63)
+- `1.32.x` - LTS release until September 2024. (MSRV 1.63)
+- `1.36.x` - LTS release until March 2025. (MSRV 1.63)
+
+Each LTS release will continue to receive backported fixes for at least a year. If you wish to use a
+fixed minor release in your project, we recommend that you use an LTS release.
 
-Each LTS release will continue to receive backported fixes for at least a year.
-If you wish to use a fixed minor release in your project, we recommend that you
-use an LTS release.
+To use a fixed minor version, you can specify the version with a tilde. For example, to specify that
+you wish to use the newest `1.32.x` patch release, you can use the following dependency
+specification:
 
-To use a fixed minor version, you can specify the version with a tilde. For
-example, to specify that you wish to use the newest `1.32.x` patch release, you
-can use the following dependency specification:
-```text
+```toml
 tokio = { version = "~1.32", features = [...] }
 ```
```

Comment:
This is rendered quite badly by github.

Reply:
Agreed. What about one of the following instead?

```toml
tokio = { version = "~1.32", features = ["..."] }
```

```toml
tokio = { version = "~1.32", features = ... }
```

Reply:
I don't have a strong preference, but the former seems more natural to me?